lib/logflare/single_tenant.ex

```diff
@@ -50,7 +93,11 @@ defmodule Logflare.SingleTenant do
   Creates an enterprise user
   """
   def create_default_user do
-    case Users.insert_user(@user_attrs) do
+    attrs =
+      @user_attrs
+      |> Map.put(:api_key, Application.get_env(:logflare, :api_key))
```

Comment:
```suggestion
    attrs = Map.put(@user_attrs, :api_key, Application.get_env(:logflare, :api_key))
```
https://github.com/rrrene/credo/issues/221